src/commands/login.ts

```diff
@@ -55,6 +55,14 @@ export async function loginCommand(accessToken: string, logSuccesMessage = true)
                 });
             });
 
+            // If we are in a CI environment, we don't open the browser because it will hang indefinitely
+            if (process.env["CI"]) {
+                log.error(
+                    "CI environment detected. Cannot open browser for authentication. Use `genezio login <token>` instead.",
+                );
+                return;
+            }
+
```

Comment:
I would do this before opening the http server.

Reply:
Nice suggestion, thanks :D